calendar_reboot.js

```diff
@@ -161,17 +188,13 @@ $(document).ready(function() {
         // and fill it with the information given in the form
         //required info are startDate and title. default to tracking 1 
         //year
+        
         var calendarTitle = $("#calendarTitle").val();
         var startDate = $("#startDate").val();
         var endDate = $("#endDate").val();
-            
         
-        var validateTitle = validateInput(calendarTitleId);
-        var validateStartDate = validateInput(startDateId);
-        var validateEndDate = validateInput(endDateId);
-        var validateDates = validateForm(startDate, endDate);
+        if (validateForm() && validateDates(startDate, endDate)) {
```

Comment:
I mean, if you have validateDates inside of validateForm, validateForm will first check that inputs were given for all input types (checking that the fields weren't left blank). Then check if End Date is indeed after Start Date. 

I guess this could all go in the same function. I have to think about it a little more.